Remotion/Web/Development.WebTesting/Utilities/ProcessUtils.cs

```diff
@@ -160,14 +249,27 @@ public static void GracefulProcessShutdown ([NotNull] Process process, TimeSpan
         // https://docs.microsoft.com/en-us/dotnet/api/system.diagnostics.process.kill#System_Diagnostics_Process_Kill
         // This exception can be swallowed safely, as we are sure that a process is associated and the process has
         // already exited, therefore the method can return at this point.
-        return;
       }
+    }
 
-      // No exception is thrown when calling .WaitForExit(..) on an already exited process.
-      if (process.WaitForExit (timeoutInMilliseconds))
-        return;
+    private static void WaitForProcessesExits (List<Process> processes, TimeSpan timeout)
```

Comment:
I think timeout should be named `sharedTimeout`, `totalTimeout` or something similar to indicate that the `timeout` does not apply to every process anew. May not be relevant here as this is a private method 🤷🏼‍♂️